src/main/java/org/apache/commons/imaging/formats/tiff/datareaders/DataInterpreterJpeg.java

```diff
@@ -0,0 +1,146 @@
+/* --------------------------------------------------------------------
+ * Copyright (C) 2023  Gary W. Lucas.
```

Comment:
Is this copyright really necessary? If you leave it in, I'll have to go check and see if we need a notice file update and whatever other legalese is involved.

Reply:
Sorry.  That copyright has absolutely no business being in the code.  My IDE inserted it and I didn't notice.  Will fix.